### PR TITLE
React framework

### PR DESCRIPTION
**Pull request by chrischo360**
Changed previous SAIL website from single page to multiple pages with a routing system. The frontend framework used is React and styling is used with Chakra UI. Routing is done by React Router.

### CODE REVIEW

**Comment by dany-fu** [comment]
Chris, this looks great. Good job on splitting up the bigger commits and the informative commit messages. Just a few high level questions: 
- I'm not seeing the Astroids game on the home page. Is it supposed to be functional in this PR?  
- Which commit has the light vs dark mode?

**Comment by chrischo360** [comment]
> Chris, this looks great. Good job on splitting up the bigger commits and the informative commit messages. Just a few high level questions:
> 
> * I'm not seeing the Astroids game on the home page. Is it supposed to be functional in this PR?
> * Which commit has the light vs dark mode?

The asteroid game should be functional on the home page and in this pull request. The light/dark mode commit is 
a7756b7.

File: index.html [hunk 1]
```diff
@@ -1,228 +1,65 @@
...
+      type="image/png"
+      href="/favicon/**.png"
+    /> -->
+
+        <link rel="apple-touch-icon" href="/favicon/s_logo.png" />
+        <link rel="manifest" href="/favicon/site.webmanifest" />
+        <meta name="theme-color" content="#000000" />
+
+        <!-- Start Single Page Apps for GitHub Pages -->
+        <script src="js/astroids.js"></script>
```
Comment (gxavier38) [inline index.html:29]: Do we still need this file if you've ported it to React?

Comment (chrischo360) [reply index.html:29]: It isn't necessary any longer

**Comment by gxavier38** [comment]
Yeah I can see the asteroids game. It's just in the background so you might have to zoom out a lot to see it. 
It looks like it's being fixed in the next branch?

**Review by gxavier38 — APPROVED** [review]

**Pull request merged**